kinetic/distribution.yaml

```diff
@@ -1,6 +1,6 @@
 %YAML 1.1
 # ROS distribution file
-# see REP 143: http://ros.org/reps/rep-0143.html
+# see REP 141: http://ros.org/reps/rep-0141.html
```

Comment:
@jack-oquin this is the only difference between the new rosdistro versions. You can edit the branch this pull-request is coming from either from (jack-oquin:bloom-ackermann_msgs-1) either in a a git client or via the github website to reference REP 143 instead of 141